### PR TITLE
Add the detailed schedule

### PR DESCRIPTION
**Pull request by CannonLock**

### CODE REVIEW

File: docs/curriculum/detailed-schedule.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+| 14:30 | 14:45 | Break                                                                            | — |
+| 14:45 | 15:30 | Lecture & demo: Software portability                                             | Christina |
+| 15:30 | 17:00 | Hands-on: Software and unfinished exercises <br> Individual consultations (some) | staff |
+
+# July 27 (Wednesday)
+
+| Start | End                            | Event                                                           | Instructor |
+|-------|--------------------------------|-----------------------------------------------------------------|------------|
+| 8:00  | 9:00 | Breakfast at Gordon Avenue Market                               | —          |
+| 8:45  | 9:15 | Walk to Computer Sciences                                       | —          |
```
Comment (CannonLock) [inline docs/curriculum/detailed-schedule.md:38]: 9:15 End to 9:30 Start

**Review by CannonLock — COMMENTED** [review]
@osg-cat One discrepancy for you to rule on

File: docs/curriculum/detailed-schedule.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+| 13:15 | 14:30 | Interactive: High Throughput Computing in action                                 | staff |
+| 14:30 | 14:45 | Break                                                                            | — |
+| 14:45 | 15:30 | Lecture & demo: Software portability                                             | Christina |
+| 15:30 | 17:00 | Hands-on: Software and unfinished exercises <br> Individual consultations (some) | staff |
+
+# July 27 (Wednesday)
+
+| Start | End                            | Event                                                           | Instructor |
+|-------|--------------------------------|-----------------------------------------------------------------|------------|
+| 8:00  | 9:00 | Breakfast at Gordon Avenue Market                               | —          |
```
Comment (CannonLock) [inline docs/curriculum/detailed-schedule.md:37]: 9:00 End to 8:45 Start

File: docs/curriculum/detailed-schedule.md (hunk 1 above)
Comment (osg-cat) [reply docs/curriculum/detailed-schedule.md:38]: Good catch! However, this one is on purpose.

**Pull request merged**